hilti/runtime/include/types/map.h

```diff
@@ -245,14 +245,37 @@ class Map : protected std::map<K, V> {
      * @return the value
      * @throw `IndexError` if `k` is not set in the map
      */
-    const V& get(const K& k) const {
+    const V& get(const K& k) const& {
```

Comment:
Oh, I don't know this, what's the difference when  overloading by `&`?

Reply:
This is an overload just for `const` lvalues as we wouldn't want to give out a ref to a value in a temporary map.